src/coordinator.rs

```diff
@@ -68,6 +69,7 @@ impl Coordinator {
 
         Ok(Self {
             hamming_distance_threshold: config.hamming_distance_threshold,
+            n_closest_distances: config.n_closest_distances,
```

Comment:
We already keep `config` in the `Coordinator` struct. There's no need for this extra copy, right?

Reply:
Yes, sir 👍 

I was just following the same pattern for `hamming_distance_threshold`. Let me remove that one as well